flycs_sdk/query_base.py

```diff
@@ -0,0 +1,38 @@
+"""This module contains base class for any "query" object (Transformation, views,...)."""
+
+from abc import ABC, abstractmethod
+from typing import Optional
+
+
+class QueryBase(ABC):
+    """Base class for any query based object."""
+
+    def __init__(
+        self, name: str, query: str, version: str, decrypt: Optional[bool] = False
+    ):
+        """Create a QueryBase object.
+
+        :param name: name of the transformation
+        :type name: str
+        :param query: SQL query
+        :type query: str
+        :param version: version of the tranformation
+        :type version: str
+        :param decrypt: if set to False, disable automatic decryption of the result of the query
```

Comment:
@christophedcpm think this should be: if set to True, disable automatic encryption of the result of the query

Reply:
This is quite intuitive, if we want that behavior the field should be called `no_decrypt` or `disable_decrypt`.